infrastructure/main.tf

```diff
@@ -17,6 +17,26 @@ data "azurerm_subnet" "core_infra_redis_subnet" {
   resource_group_name  = "core-infra-${var.env}"
 }
 
+module "submit-your-appeal-frontend" {
```

Comment:
why is this here?

Reply:
sorry I think by mistake I've added this, you are right its not needed for AKS. Let me removed it